app/templates/tasklist.html

```diff
@@ -6,7 +6,7 @@
 {% block beforeContent %}
     <a href="{{ url_for('account_routes.dashboard') }}" class="govuk-back-link">Back to your applications</a>
 {% endblock %}
-{% block pageTitle %}{% trans %}Task List{% endtrans %} - {{ service_title }}{% endblock %}
+{% set pageHeading %}{% trans %}Task List{% endtrans %}{% endset %}
```

Comment:
should this be used on page?

Reply:
There is an inconsistency in this case, Task List was set as the title which is different than the `H1` so I went with Task List as that's more specific.